index.js

```diff
@@ -5,6 +5,18 @@ function fixCapitalisedPlural(fn) {
   };
 }
 
+function fixChangePlural(fn) {
+  return function(str) {
+    const matches = str.match(/[A-Z_][a-z0-9]*$/);
+    const index = matches ? matches.index : 0;
+    const hasUnderscore = matches && matches[0].startsWith("_");
```

Comment:
Slightly more optimal:
```suggestion
    const hasUnderscore = str[index] === "_";
```